src/foundry/foundry.js/pixi/polygons/sourcePolygon.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * An extension of the base PIXI.Polygon class designed for polygons which are constructed as a radius around
+ * some central origin.
+ */
+declare class SourcePolygon extends PIXI.Polygon {
+  constructor(x: number, y: number, radius: number, ...points: ({ x: number; y: number }[] | number[])[]);
```

Comment:
Is it intentional that you replaced `PIXI.IPoint` with `{ x: number, y: number }`?

Reply:
Yes, it’s never called with a `PIXI.IPoint` In foundry. I also verified that using `{ x: number, y: number }` is fine. I actually looked in the implementation in PIXI. They only use the `x` and `y` properties.